src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+    },
```

Comment:
check task description: each `operate` call takes a callback and a number and sets the appropriate value to the `result` property.
separate calculate functions shouldn't know anything about the object or changing some of calculator properties
they just should accept needed numbers and provide needed result